rigex-front-end/src/components/NavBar.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Button from '@material-ui/core/Button';
+import logo from "../assets/rigex-logo-small.svg";
+import search from "../assets/search.svg";
+import account from "../assets/account.svg";
+
+const useStyles = makeStyles(() => ({
+  navBar: {
+    flexGrow: 1,
+    background: 'white',
```

Comment:
Let's use css variables for all colors. The color variables should be stored in a file and imported into where you use them in your styles. You may have to do some googling for this